src/plugins/ovp/components/jwplayer-media-fields/index.jsx

```diff
@@ -0,0 +1,39 @@
+import { Card, CardBody, CardHeader } from '@triniti/admin-ui-plugin/components';
+import { Field } from 'redux-form';
+import CheckboxField from '@triniti/cms/components/checkbox-field';
+import PropTypes from 'prop-types';
+import React from 'react';
+import TextField from '@triniti/cms/components/text-field';
+
+const JwplayerMediaFields = ({ isEditMode }) => (
+  <Card>
+    <CardHeader>JW Player</CardHeader>
+    <CardBody indent>
+      <Field
+        component={TextField}
+        label="JW Player Media ID"
+        name="jwplayerMediaId"
+        placeholder="enter jw player media id..."
+        readOnly={!isEditMode}
+      />
+      <Field
```

Comment:
@efjacobson - I think we don't need to show in the UI. Users that have access to the `raw` tab can get this info there.

Reply:
@arunkarnati it mirrors kaltura_synced_at

Reply:
@efjacobson - gotcha, I'm ok with it.